CurvedSegment.py

```diff
@@ -148,7 +148,7 @@ def makeRibs(self, fp):
         
         
     def rescaleRibs(self, fp, ribs):
-        if (fp.makeSurface or fp.makeSolid) and fp.Path is None:
+        if (fp.makeSurface or fp.makeSolid) and fp.Path is None and abs(fp.ActualTwist)<=epsilon:
             start = 1
```

Comment:
the original Shape2 is obviously not twisted, so if twist is introduced and not a multiple of 360° this causes a big issue when the second to last rib is lofted with Shape2 -- solution: close with a twisted copy instead - same as with the Path